pandas/core/ops.py

```diff
@@ -990,6 +991,26 @@ def _construct_divmod_result(left, result, index, name, dtype):
     )
 
 
+def dispatch_to_extension_op(left, right, op_name):
+    """
+    Assume that left is a Series backed by an ExtensionArray,
+    apply the operator defined by op_name.
+    """
+
+    method = getattr(left.values, op_name, None)
+    if method is not None:
+        res_values = method(right)
+    if method is None or res_values is NotImplemented:
+        msg = "ExtensionArray invalid operation {opn} between {one} and {two}"
+        raise TypeError(msg.format(opn=op_name,
```

Comment:
given my comment above, this should not be needed (as the ExtensionArray op will raise the appropriate error)

Reply:
Yes, that makes sense.  I changed it.